tests/ternaries/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -273,3 +358,55 @@ const obj5 = conditionIsTruthy
       };
 
 `;
+
+exports[`test.js 3`] = `
+const obj0 = conditionIsTruthy ? shortThing : otherShortThing
+
+const obj1 = conditionIsTruthy ? { some: 'long', object: 'with', lots: 'of', stuff } : shortThing
+
+const obj2 = conditionIsTruthy ? shortThing : { some: 'long', object: 'with', lots: 'of', stuff }
+
+const obj3 = conditionIsTruthy ? { some: 'eeeeeeeeeeeeven looooooooooooooooooooooooooooooonger', object: 'with', lots: 'of', stuff } : shortThing
+
+const obj4 = conditionIsTruthy ? shortThing : { some: 'eeeeeeeeeeeeven looooooooooooooooooooooooooooooonger', object: 'with', lots: 'of', stuff }
+
+const obj5 = conditionIsTruthy ? { some: 'long', object: 'with', lots: 'of', stuff } : { some: 'eeeeeeeeeeeeven looooooooooooooooooooooooooooooonger', object: 'with', lots: 'of', stuff }
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+const obj0 = conditionIsTruthy ? shortThing : otherShortThing;
+
+const obj1 = conditionIsTruthy
+	? { some: "long", object: "with", lots: "of", stuff }
+	: shortThing;
+
+const obj2 = conditionIsTruthy
+	? shortThing
+	: { some: "long", object: "with", lots: "of", stuff };
+
+const obj3 = conditionIsTruthy
+	? {
+			some: "eeeeeeeeeeeeven looooooooooooooooooooooooooooooonger",
```

Comment:
isnt this indentation too big? opening and closing curly bracket dont align cause of this

Reply:
That's unrelated to this bug (it was already this way)

@ikatyang opened #3277 to address it